### PR TITLE
[CA-1325] Update readme for integration tests

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]

File: integration-tests/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ See also: [Bueller](Bueller.md) - service for invoking tests
 
 1. Start by making sure you're running the correct versions of Node and Yarn, as described in the [root README](../README.md).
 2. Run `yarn install` in this directory.
-3. To set up an environment, run `node scripts/initializeEnvironment.js`.
+3. To set up an environment (e.g., dev, alpha, prod. Not your local environment), run `node scripts/initializeEnvironment.js`.
```
Comment (gpolumbo-broad) [inline integration-tests/README.md]: I would extract this into its whole own section.  I think there should be one section for "here are instructions for you to set up your local environment to be able to run the integration tests" and a separate section for "here's what you need to do to configure a live Terra environment to be used by the integration test suite".

**Review by gpolumbo-broad — CHANGES_REQUESTED** [review]

File: integration-tests/README.md (hunk 1 above)
Comment (s-rubenstein) [reply integration-tests/README.md]: Done!

**Review by breilly2 — APPROVED** [review]
👍   Thanks for clarifying this!

**Comment by zarsky-broad** [comment]
Merging with failing status due to known issue, no code changes in pr.

**Pull request merged**